sops.go

```diff
@@ -318,6 +318,13 @@ func (tree Tree) Encrypt(key []byte, cipher Cipher) (string, error) {
 				}
 			}
 		}
+		if tree.Metadata.EncryptedSelector != "" {
```

Comment:
This is good, but I think a better approach would be to convert the selector to a `[]string` (in `main.go`, so the SOPS API itself would take the `[]string`, and the command line will handle conversion) and then compare the slices. It saves us some computations we currently do for every decryption, and more importantly would provide a cleaner API to those using SOPS as a Go library.

What do you think?